source/common/http/codec_client.h

```diff
@@ -72,6 +72,13 @@ class CodecClient : protected Logger::Loggable<Logger::Id::client>,
    */
   bool isHalfCloseEnabled() { return connection_->isHalfCloseEnabled(); }
 
+  /**
+   * Initialize all of the installed read filters on the underlying connection.
+   * This effectively calls onNewConnection() on each of them.
+   * @return true if read filters were initialized successfully, otherwise false.
+   */
+  bool initializeReadFilters() { return connection_->initializeReadFilters(); }
```

Comment:
hm, this seems like a bug fix. I'd be inclined to runtime guard and call out in release notes

Reply:
I've never added a runtime guard before, can you point me to a representative commit I can look for inspiration?

Reply:
Added runtime guard `initialize_upstream_filters` and called this out in release notes.